ncats_webd/cybex_queries.py

```diff
@@ -245,6 +245,7 @@ def csv_get_open_tickets(db, ticket_severity):
             index=False,
             date_format="%Y-%m-%d %H:%M:%S",
             columns=[
+                "id",
```

Comment:
Use correct field name:
```suggestion
                "_id",
```

Reply:
updated

Reply:
When responding to suggestions on GitHub, we typically do one the following:
1. _(Strongly preferred)_ Use the ["Commit suggestion" feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) in the GitHub web UI.  This has the advantage of automatically resolving the conversation and giving co-commit credit to the person who made the suggestion.  It also helpfully lets you batch related suggestions into a single commit.
1. If you don't want to use the "Commit suggestion" feature (rare, but sometimes there's a good reason to not use it, such as when someone provides a partially-correct suggestion, but you want to add more), then we ask that you reference the relevant commit in your reply (in this case: https://github.com/cisagov/ncats-webd/pull/32/commits/8f902f5d3bea48df17386cd5c572c12292b7bbf8).  It's also polite to give co-commit credit to the person who made the suggestion, but that's not required.

Additionally, we strongly encourage developers to follow ["The seven rules of a great Git commit message"](https://cbea.ms/git-commit/#seven-rules).  In the case of https://github.com/cisagov/ncats-webd/pull/32/commits/8f902f5d3bea48df17386cd5c572c12292b7bbf8, we would like to have seen the commit message (subject line) in the [imperative mood](https://cbea.ms/git-commit/#imperative), i.e. `Update field name` (instead of "updated").  This really makes the git commit log read more naturally.  It's a little weird to get used to at first, but once you do, it looks strange to see commit subject line messages written any other way.

Reply:
@rc-raizen - Neither https://github.com/cisagov/ncats-webd/pull/32/commits/8f902f5d3bea48df17386cd5c572c12292b7bbf8 nor https://github.com/cisagov/ncats-webd/pull/32/commits/91063d7eb1a2d4205d70c2cd29ba1b2e71cbf908 addresses the changes I suggested.

Reply:
Just as a documentation add when @dav3r refers to "co-commit credit" he is referring to leveraging co-author functionality as described in this [GitHub documentation](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors).